File: ucare_backend/src/main/java/com/douzone/ucare/service/TimeService.java
package com.douzone.ucare.service;

import java.util.List;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

import com.douzone.ucare.repository.TimeRepository;
import com.douzone.ucare.vo.ReceiptVo;
import com.douzone.ucare.vo.TimeVo;

@Service
public class TimeService {
	
	@Autowired
	private TimeRepository timeRepository;

	public int update(String date) {
		return timeRepository.update(date);
	}

	public int updateByCancel(ReceiptVo data) {
		return timeRepository.updateByCancel(data);
	}
	
	public List<TimeVo> retrieveTime(String date) {
		return timeRepository.retrieveTime(date);
	}
	
	public int updateTime(TimeVo data) {
		return timeRepository.updateTime(data);
	}
}

File: ucare_frontend/src/service/timeService.js
import axios from 'axios';
import { TIME_API_BASE_URL } from './urlConfig'

class timeService {
  update(date){
    return axios.put(TIME_API_BASE_URL + '/update/' + date, null, { headers: { Authorization: localStorage.getItem("authorization") }})
  }
  updateByCancel(data){
    return axios.put(TIME_API_BASE_URL + '/updateByCancel', data, { headers: { Authorization: localStorage.getItem("authorization") }})
  }
  retrieveAll(date){
    return axios.get(TIME_API_BASE_URL + '/retrieveAll/' + date, { headers: { Authorization: localStorage.getItem("authorization") }});
  }
  updateTime(date){
    return axios.put(TIME_API_BASE_URL + '/updateTime', date, { headers: { Authorization: localStorage.getItem("authorization") }})
  }
  updateDelete(date){
    return axios.put(TIME_API_BASE_URL + '/updateDelete', date, { headers: { Authorization: localStorage.getItem("authorization") }})
  }
}

export default new timeService();